src/dbf-file.ts

```diff
@@ -1,11 +1,11 @@
 import * as assert from 'assert';
 import * as iconv from 'iconv-lite';
-import {extname} from 'path';
-import {FieldDescriptor, validateFieldDescriptor} from './field-descriptor';
-import {isValidFileVersion} from './file-version';
-import {CreateOptions, Encoding, normaliseCreateOptions, normaliseOpenOptions, OpenOptions} from './options';
-import {close, open, read, stat, write} from './utils';
-import {createDate, format8CharDate, formatVfpDateTime, parseVfpDateTime, parse8CharDate} from './utils';
+import { extname } from 'path';
```

Comment:
can you please remove all the whitespace changes from the PR and only change the code where you are adding new functionality?

Reply:
![test_cases](https://user-images.githubusercontent.com/8259729/143683159-69270878-2fd2-43c5-8aa2-f9606752d2c2.PNG)

Do you know why complied test files are created under `test/test` directory? Ideally they should be created under `test` directory.

Reply:
I am just running command `npm run build` to compile the source code.

Reply:
I'm not sure how you are getting that. I can't reproduce the above screenshot either locally or using codespaces. Running `npm install` then `npm run build` creates the built test files under `dist/test` as per the `tsconfig` and npm script.